### PR TITLE
Fixed `F`'s `asc` and `desc` args

### PR DESCRIPTION
**Pull request by g-as**
# I have made things!

Fixed `asc` and `desc` signature by removing `expression`.

I also updated `BaseExpression`'s `asc` and `desc` (they are the same).

```python

class OrderBy(Expression):
    ...

    def __init__(self, expression, descending=False, nulls_first=False, nulls_last=False):
        ...

    ...

@deconstructible
class F(Combinable):
    """An object capable of resolving references to existing query objects."""

    ...

    def asc(self, **kwargs):
        return OrderBy(self, **kwargs)

    def desc(self, **kwargs):
        return OrderBy(self, descending=True, **kwargs)
        
    ...
```

### CODE REVIEW

File: django-stubs/db/models/expressions.pyi [hunk 1]
```diff
@@ -121,15 +132,13 @@ class F(Combinable):
     def asc(
```
Comment (sobolevn) [inline django-stubs/db/models/expressions.pyi:132]: Do we still need these method? They are duplicating the `BaseExpression` ones.
Or is it django-version specific?

Comment (g-as) [reply django-stubs/db/models/expressions.pyi:132]: `F` doesn't inherit from `BaseExpression` but from `class Combinable:`

Comment (sobolevn) [reply django-stubs/db/models/expressions.pyi:132]: Oh, I am sorry :)

Comment (g-as) [reply django-stubs/db/models/expressions.pyi:132]: No worries.

Comment (sobolevn) [reply django-stubs/db/models/expressions.pyi:132]: Source: https://github.com/django/django/blob/09397f5cfade93c2e79a391ded7ab6b01e51730e/django/db/models/expressions.py#L818-L819

**Review by sobolevn — APPROVED** [review]

**Comment by sobolevn** [comment]
Thank you! 👍

**Comment by g-as** [comment]
Cheers.

**Pull request merged**